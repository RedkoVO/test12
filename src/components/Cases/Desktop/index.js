import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import MenuNavigation from '../../MenuNavigation'
import Header from '../../../containers/Header/AsyncHeader'
import Footer from '../../Footer/Desktop/'

import styles from './styles'

const Cases = ({ classes, handleLogout }) => (
  <div className="mainWrapperTrade">
    <div className="containerTradeWrap">
      <div className="containerTrade">
        <header className={classes.headerSkins}>
          <MenuNavigation page="dark" handleLogout={handleLogout} />
          <Header type="catalog" />
        </header>

        {/* <main className={classes.tradeMain}>

        </main> */}
      </div>
    </div>

    <Footer />
  </div>
)

Cases.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(Cases)
