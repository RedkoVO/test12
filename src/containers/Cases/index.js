import compose from 'recompose/compose'
import { pure } from 'recompose'

import AsyncCasesDesktop from '../../components/Cases/Desktop/AsyncCasesDesktop'

export default compose(pure)(AsyncCasesDesktop)
