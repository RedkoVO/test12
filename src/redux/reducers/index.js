import { combineReducers } from 'redux'
import { reducer as form } from 'redux-form'
import auth from './auth'
import balance from './balance'

export default combineReducers({
  form,
  auth,
  balance
})