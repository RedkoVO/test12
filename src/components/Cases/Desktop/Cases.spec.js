import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import { Provider } from 'react-redux'
import { createStore } from 'redux'

import Cases from './'

import CaseXIcon from '../../../assets/images/caseX.png'

const tmpCases = [
  {
    id: 1,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 2,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 3,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 4,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 5,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 6,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 7,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 8,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 9,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 10,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 11,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  },
  {
    id: 12,
    img: CaseXIcon,
    title: 'Hello, buddy!',
    cost: '$34.20',
    game: 'CS:GO'
  }
]
const store = createStore(a => a, {})

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <MemoryRouter keyLength={0}>
          <Cases cases={tmpCases} />
        </MemoryRouter>
      </Provider>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
