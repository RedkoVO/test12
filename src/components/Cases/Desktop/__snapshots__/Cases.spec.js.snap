// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mainWrapperTrade"
>
  <div
    className="Cases-containerTradeWrap-1"
  >
    <div
      className="Cases-containerTrade-2"
    >
      <header
        className="Cases-headerSkins-3"
      >
        <ul
          className="MenuNavigation-root-5 dark"
        >
          <li
            className={undefined}
          >
            <div
              className="MenuNavigation-logout-14"
              onClick={[Function]}
            >
              Logout
            </div>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-dashboard-8"
              />
              <br />
               Dashboard
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-walletIcon-9"
              />
              <br />
               Wallet
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-shop-10"
              />
              <br />
               Shop
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/games"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-games-11"
              />
              <br />
               Games
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-friends-12"
              />
              <br />
               Friends
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-settings-13"
              />
              <br />
               Settings
            </a>
          </li>
        </ul>
      </header>
      <main
        className="Cases-cases-4"
      >
        <div
          className="CasesFilter-root-15"
        >
          <input
            className="CasesFilter-casesFilterSearch-16"
            placeholder="Search"
            type="text"
          />
          <div
            className="CasesFilter-casesFilterOptions-17"
          >
            <div
              className="CasesFilter-casesFilterType-18"
            >
              <div
                className="CasesFilter-casesFilterItem-19"
              >
                less than $15&lt;
              </div>
              <div
                className="CasesFilter-casesFilterItem-19"
              >
                $15-30
              </div>
              <div
                className="CasesFilter-casesFilterItem-19 CasesFilter-selectedItem-21"
              >
                 $30-60
              </div>
              <div
                className="CasesFilter-casesFilterItem-19"
              >
                $60-90
              </div>
              <div
                className="CasesFilter-casesFilterItem-19"
              >
                less than $15&lt;
              </div>
            </div>
            <div
              className="CasesFilter-casesFilterType-18"
            >
              <div
                className="CasesFilter-casesFilterItem-19"
              >
                CS:GO
              </div>
              <div
                className="CasesFilter-casesFilterItem-19"
              >
                Dota 2
              </div>
              <div
                className="CasesFilter-casesFilterItem-19 CasesFilter-selectedItem-21"
              >
                Starcraft 2
              </div>
              <div
                className="CasesFilter-casesFilterItem-19"
              >
                 Maincraft
              </div>
              <div
                className="CasesFilter-casesFilterItem-19"
              >
                Rainbow Six
              </div>
            </div>
          </div>
          <div
            className="CasesFilter-casesFilterBtn-22"
          />
        </div>
        <div
          className="CasesList-root-23"
        >
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
          <div
            className="CasesList-casesListItem-24"
          >
            <div
              className="CasesList-casesListItemImg-25"
            >
              <img
                alt=""
                src="caseX.png"
              />
            </div>
            <div
              className="CasesList-casesListItemName-26"
            >
              Case: 
              <span>
                Hello, buddy!
              </span>
            </div>
            <div
              className="CasesList-casesListItemPrise-27"
            >
              Open for 
              <span>
                $34.20
              </span>
            </div>
            <div
              className="CasesList-casesListItemGame-28"
            >
              Case: 
              <span>
                CS:GO
              </span>
            </div>
            <div
              className="CasesList-casesListItemMarker-29"
            />
          </div>
        </div>
      </main>
    </div>
  </div>
  <footer
    className="Footer-root-30"
  >
    <ul
      className="Footer-footerNav-31"
    >
      <li>
        <a
          className="Footer-footerNavItem-32"
          href="/"
          onClick={[Function]}
        >
          Dashboard
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-32"
          href="/"
          onClick={[Function]}
        >
          Wallet
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-32"
          href="/"
          onClick={[Function]}
        >
          Shop
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-32"
          href="/games"
          onClick={[Function]}
        >
          Games
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-32"
          href="/"
          onClick={[Function]}
        >
          Friends
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-32"
          href="/"
          onClick={[Function]}
        >
          Settings
        </a>
      </li>
    </ul>
    <span
      className={undefined}
    >
      Vault. All Rights Reserved 2018
    </span>
  </footer>
</div>
`;
